docs/DeveloperGuide.md

```diff
@@ -133,6 +155,50 @@ Guarantees: Editing of skill or achievement is successful
   1a1. InternHunter displays an error message and informs the user of the valid input format. </br>
   Use case resumes from step 1.
 
+
+### Use case: UC14 - Get help
+
+#### MSS
+
+Guarantees: User will get directions to the user guide
+
+MSS:
+1.  User requests for help.
+2.  InternHunter displays help message directing user to user guide. </br>
+    Use case ends.
+
+
+### Use case: UC15 - Exit 
+
+#### MSS
+
+Precondition: System is running
+Guarantees: System will stop running
```

Comment:
I don't think there should be a guarantee here that the system stops running since extension at 2a implies that use case ends when user decides to cancel the confirmation

Reply:
sounds good!